olive/systems/system_config.py

```diff
@@ -15,6 +15,16 @@
 from olive.systems.common import AzureMLDockerConfig, LocalDockerConfig, SystemType
 
 
+class VCConfig(ConfigBase):
+    instance_type: str
+    interactive: bool = False
+    imageVersion: str = ""
+    slaTier: str = "Standard"
+    priority: str = "Medium"
+    tensorboardLogDirectory: str = "/scratch/tensorboard_logs"
```

Comment:
default to None or required instead of the hardcode value?

Reply:
I'm testing the None value for these attributes now. If it supports None and the API can map a default value, I will update this logic.